server/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const app = express()
+const port = 3000
+const path = require('path')
+
+app.get('/', (req, res) => { 
```

Comment:
Did you consider `app.all(express.static...` ).  Without doing this, won't you have to set up separate get request handlers for all js and css files (e.g. bundle.js)?

Reply:
Hi ronny! Yes, I agree using express.static is a much better choice for static files. This was simply bare-bones set up so I can start expanding. I'll definitely add express.static in the next steps. Thanks!